src/domain/safe/safe.repository.ts

```diff
@@ -538,15 +538,11 @@ export class SafeRepository implements ISafeRepository {
       if (!this.isTrustedDelegateCallEnabled) {
         throw new UnprocessableEntityException('Delegate call is disabled');
       }
-      try {
-        const contract = await this.contractsRepository.getContract({
-          chainId: args.chainId,
-          contractAddress: args.proposeTransactionDto.to,
-        });
-        if (!contract.trustedForDelegateCall) {
-          throw new Error('Delegate call is disabled');
-        }
-      } catch {
+      const contract = await this.contractsRepository.getContract({
+        chainId: args.chainId,
+        contractAddress: args.proposeTransactionDto.to,
+      });
+      if (contract && !contract.trustedForDelegateCall) {
```

Comment:
If a `contract` doesn't exist, this will allow delegate calls.

Reply:
If the contract does not exist, I think the flow should stop there and the service would throw an error. So no further call to the TXS propose endpoint. But given this is unclear, and it's just a detail (the error message), I'm closing the PR to avoid distractions.